packages/core/src/lib/light_push/light_push.ts

```diff
@@ -19,19 +19,17 @@ import { Uint8ArrayList } from "uint8arraylist";
 
 import { BaseProtocol } from "../base_protocol.js";
 
-import { PushRpc } from "./push_rpc.js";
-import { isRLNResponseError } from "./utils.js";
+import { PushRpcV2 } from "./push_rpc_v2.js";
+import { mapInfoToProtocolError } from "./utils.js";
 
 const log = new Logger("light-push");
 
 export const LightPushCodec = "/vac/waku/lightpush/2.0.0-beta1";
-export { PushResponse };
 
-type PreparePushMessageResult = ThisOrThat<"query", PushRpc>;
+export const LightPushCodecV2 = LightPushCodec;
+
+type PreparePushMessageResult = ThisOrThat<"query", PushRpcV2>;
```

Comment:
it doesn't seem to be used at all

```suggestion
type PreparePushMessageResult = ThisOrThat<"query", PushRpcV2>;
```